### PR TITLE
 [Fix #4190] add example for Naming/AsciiIdentifier 

### PR DESCRIPTION
**Pull request by YutaGoto**
Add example for Naming/AsciiIdentifier 

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] All tests(`rake spec`) are passing.
* [x] The new code doesn't generate RuboCop offenses that are checked by `rake internal_investigation`.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by pocke** [comment]
We have a cop that is detect non-ascii comment. So RuboCop adds offenses for the examples.

```
$ bundle exec rake internal_investigation
Running RuboCop...
Inspecting 1019 files
.....................................................................................................................................................................................................C.....................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................

Offenses:

lib/rubocop/cop/naming/ascii_identifiers.rb:10:17: C: Use only ascii symbols in comments.
      #     def καλημερα
                ^^^^^^^^
lib/rubocop/cop/naming/ascii_identifiers.rb:13:17: C: Use only ascii symbols in comments.
      #     def こんにちは
                ^^^^^
lib/rubocop/cop/naming/ascii_identifiers.rb:16:23: C: Use only ascii symbols in comments.
      #     def hello_🍣
                      ^

1019 files inspected, 3 offenses detected
```


So please add disable comment for the offenses. For example: https://github.com/bbatsov/rubocop/blob/7936abec888d8a3bfe0ca8aba39a8b24346f4f4e/lib/rubocop/cop/style/ascii_comments.rb#L3-L16

File: lib/rubocop/cop/naming/ascii_identifiers.rb [hunk 1]
```diff
@@ -4,6 +4,19 @@ module RuboCop
   module Cop
     module Naming
       # This cop checks for non-ascii characters in identifier names.
+      #
+      # @example
+      #     # bad
+      #     def καλημερα
```
Comment (pocke) [inline lib/rubocop/cop/naming/ascii_identifiers.rb]: The examples are not complete ruby code. I think the example should include `end` also.

**Comment by YutaGoto** [comment]
@pocke Thanks your reviews. I added `# rubocop:disable Style/AsciiComments` and RuboCop said no offenses on my local env.

**Comment by bbatsov** [comment]
You should probably add examples with local variables and params that are named inappropriately. Right now the examples can mislead someone into thinking the cop checks only method names. You can get some inspiration from the spec of this cop.

**Comment by YutaGoto** [comment]
@bbatsov Thanks your feedback! I added local variables example and params example.

File: lib/rubocop/cop/naming/ascii_identifiers.rb [hunk 2]
```diff
@@ -3,7 +3,39 @@
 module RuboCop
   module Cop
     module Naming
+      # rubocop:disable Style/AsciiComments
```
Comment (bbatsov) [inline lib/rubocop/cop/naming/ascii_identifiers.rb]: Move this to the beginning of the class and remove the enable, so they won't appear in the manual. Probably we should fix the generator to simply exclude those lines, as we've had this problem a few times recently.

**Comment by bbatsov** [comment]
👍

**Pull request merged**